app/views/shared/provider/navigation/_vertical_nav.html.slim

```diff
@@ -70,7 +70,7 @@ nav.vertical-nav#mainmenu
             nav.classList.add('hover-secondary-nav-pf');
             navItems[me].classList.add('is-hover');
 
-            if (itemVisible) {
+            if (itemVisible !== null) {
```

Comment:
is it possible that `itemVisible` would be `undefined`, right? that'd be breaking the flow

Reply:
It should not be possible, its values are `null | number` by design.